uaclient/util.py

```diff
@@ -65,16 +61,17 @@ def encode_text(text, encoding='utf-8'):
     return text.encode(encoding)
 
 
-def is_container():
+def is_container(run_path='/run'):
```

Comment:
This shouldn't block this landing, but what is `run_path` intended to be used for?  It doesn't look like it's utilised anywhere in this PR.

Reply:
run_path is used when we fall through and don't have systemd-detect-virt utility

Reply:
Sorry, I didn't mean `run_path` as a local variable, I meant `run_path` as a parameter; why does it need to be modifiable by callers (and if it does need to be, why aren't any of them doing so now)?

Reply:
ahh right, it was intended to allow me to add a unit test that seeded tmpdir fixture in there, but I never committed a new unit test for it.